src/QirRuntime/test/QIR-static/qir-test-math.cpp

```diff
@@ -0,0 +1,16 @@
+#include <cmath>
+
+#include "catch.hpp"
+
+extern "C" double __quantum__qis__sqrt__body(double);   // NOLINT
```

Comment:
I'm not sure I like this direct testing of the bridge. While it has some value, I'd prefer to add a qs file that we'd use to generate QIR and then test that. This way we'll also check that our expectation of the generated signatures matches the compiler's output.

Reply:
Moved the test to the .qs file.